lua/nordic/tests/options.lua

```diff
@@ -20,6 +20,18 @@ config.on_palette = function(palette)
     return palette
 end
 
+config.on_highlight = function(highlights, palette)
+    highlights.TelescopePromptTitle = {
+        fg = palette.red.bright,
+        bg = palette.green.base,
+        italic = true,
+        underline = true,
+        sp = palette.yellow.dim,
+        undercurl = false
+    }
+    return highlights
```

Comment:
Remember we removed the return value.

Reply:
Should probably remove it from `on_palette` as well.

Reply:
I did not notice that. However, I think it would be good to leave a copy like that because I want to make sure we don't break configs that do return the value. What do you think?


I am going to refactor the whole test setup soon, so maybe put deprecation tests in their own place.

Either way, I will fix it for now and add it back to a different PR if we decide we want something like that.

Reply:
It shouldn't break existing configs I think, should be fine to remove.